tests/cupyx_tests/scipy_tests/special_tests/test_logsumexp.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+import numpy
+
+import cupy
+from cupy import testing
+import cupyx.scipy.special  # NOQA
+
+import scipy.special  # NOQA
```

Comment:
SciPy is not necessarily installed when test runs. Please follow other cupyx.scipy tests in case scipy is not installed.